php/unic/application/template/errors/403.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```

Comment:
is it not an unused file @rajkumardusad ?

Reply:
Its error template like 404, 500 etc.
But we can remove it.